packages/config/src/configs/karura.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AssetMinBuilder,
+  BalanceBuilder,
+  ExtrinsicBuilder,
+  FeeBuilder,
+} from '@moonbeam-network/xcm-builder';
+import { ausd, kar, movr } from '../assets';
+import { karura, moonriver } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const karuraConfig = new ChainConfig({
+  assets: [
+    new AssetConfig({
+      asset: kar,
+      balance: BalanceBuilder().system().account(),
+      destination: moonriver,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: kar,
+      },
+      extrinsic: ExtrinsicBuilder().xTokens().transfer(),
+    }),
+    new AssetConfig({
+      asset: ausd,
+      balance: BalanceBuilder().tokens().accounts(),
+      destination: moonriver,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: kar,
```

Comment:
Same as with acala. `ausd` transactions are paid by `ausd` itself in the destination, not `kar`